lib/wp-connector.rb

```diff
@@ -1,5 +1,6 @@
 require 'wp-connector/version'
 require 'wp-connector/railtie' if defined?(Rails)
+require 'exceptions'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.